e2e/licenses/airgap-license.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: kots.io/v1beta1
+kind: License
+metadata:
+  name: githubsecretairgapcitestcustomer
+spec:
+  appSlug: embedded-cluster-smoke-test-staging-app
+  channelID: 2eAqMYG1IEtX8cwpaO1kgNV6EB3
+  channelName: CI-airgap
+  channels:
+  - channelID: 2eAqMYG1IEtX8cwpaO1kgNV6EB3
+    channelName: CI-airgap
+    channelSlug: ci-airgap
+    endpoint: https://staging.replicated.app
+    isDefault: true
+    replicatedProxyDomain: proxy.staging.replicated.com
+  customerName: GitHub Secret Airgap CI Test Customer
+  endpoint: https://staging.replicated.app
```

Comment:
shouldn't these be the custom domain?

Reply:
only if configured on the channel, which we turned off intentionally to make sure we use the domains embedded into the binary